CockpitPlugin/src/main/java/com/polidea/cockpitplugin/generator/DebugCockpitGenerator.kt

```diff
@@ -2,15 +2,16 @@ package com.polidea.cockpitplugin.generator
 
 import com.polidea.cockpit.core.CockpitParam
 import com.polidea.cockpit.core.type.CockpitAction
+import com.polidea.cockpit.core.type.CockpitColor
 import com.polidea.cockpit.core.type.CockpitListType
-import com.squareup.javapoet.MethodSpec
-import com.squareup.javapoet.ParameterSpec
+import com.squareup.javapoet.*
 import java.io.File
 import javax.lang.model.element.Modifier
 
-
 internal class DebugCockpitGenerator : BaseCockpitGenerator() {
 
+    private var shouldGenerateColorFields = false
```

Comment:
I think we should generate `colorListenerMap` and `cockpitColorMapper` only if there is at least one color parameter. Otherwise, those fields will be unused.